torchtitan/config_manager.py

```diff
@@ -746,20 +746,17 @@ def __init__(self):
             help="The minimum number of FT replica for each step.",
         )
 
-        # I'm not particularly fond of this. Users can choose to write their own wrapper
-        # module and import torchtitan training loop and execute it, which look cleaner.
-        # One reason to provide this option is to allow users to use the existing run script.
-        # While the script is pretty trivial now, we may add more logic when integrating
-        # with TorchFT.
-        # This option is subject to change and may be deleted in the future.
         self.parser.add_argument(
             "--experimental.custom_model_path",
             type=str,
             default="",
             help="""
-                This option allows importing an external module from a custom path.
```

Comment:
should this type be changed from str to List[str]?  we could support importing several custom things, i guess?  not sure if thats important, we can skip

Reply:
I prefer to let users handle other import by themselves for simplicity. More typing in the command line higher chances of typo. Just my personal opinion.

Reply:
Yea if possible imports should always be done from inside the code. But also it seems plausible that you could have multiple external packages you want to load from. Don't have a strong preference for this

Reply:
@wconstab Makes sense. Let me land this PR first. If we have this use case, we can extend it to accept multiple paths.